From 778a08d00ef4426d177f18a198fe5735fcbe6949 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Thu, 24 Jan 2019 07:15:25 +0100
Subject: [PATCH] base-files: drop sysupgrade

We have our own method of upgrading system and having sysupgrade in
system is dangerous.
---
 package/base-files/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/base-files/Makefile b/package/base-files/Makefile
index 4adfa7d5b8..66d68c6829 100644
--- a/package/base-files/Makefile
+++ b/package/base-files/Makefile
@@ -183,6 +183,7 @@ define Package/base-files/install
 				echo "$$$${conffile##$(1)}" >> $(1)/CONTROL/conffiles; \
 		fi \
 	done
+	rm -f /sbin/sysupgrade
 
 	$(if $(CONFIG_INCLUDE_CONFIG), \
 		echo -e "# Build configuration for board $(BOARD)/$(SUBTARGET)/$(PROFILE)\n" >$(1)/etc/build.config; \
-- 
2.19.2

